src/activities/drawnumber/Drawnumber.qml

```diff
@@ -212,6 +204,10 @@ ActivityBase {
 
             onPressed: {
                 checkPoints(touchPoints)
+                // to play upon clicking
+                if(mode == "drawletters"){
+                activity.audioEffects.play(Activity.url + "buttonclick."+ApplicationInfo.CompressedAudio)
```

Comment:
Yes, we can play sound in all 3 sections :- drawletters,drawnumber,clickanddraw.